packages/cdk8s-cli/lib/import/base.ts

```diff
@@ -56,7 +56,7 @@ export abstract class ImportBase {
         });
 
         const pacmak = require.resolve('jsii-pacmak/bin/jsii-pacmak');
-        await shell(pacmak, [ '--target', options.targetLanguage, '--code-only' ]);
```

Comment:
Since both `jsii` and `pacmak` are node programs, I rather we changed `shell` to use `spawnFile`:

```
spawnFile(process.execPath, [ pacmak, '--target', options.targetLanguage, '--code-only' ]);
```

See https://github.com/awslabs/cdk8s/blob/0bf8dfd6a8346b8b2f0e33eac0a2ed32d384c387/packages/cdk8s/lib/yaml.ts#L74 as an example.

Reply:
Works for me! I'll update it

Reply:
Hey, I am realizing this PR is also going to be superseded by my extraction of jsii logic into a new library (you can check it out in https://github.com/eladb/jsii-srcmak)